### DISCUSSION TITLE
Monitoring Epoptes Clients from a Manjaro 18.0.4 Machine

### DISCUSSION THREAD

**Discussion by seconddayout** [epoptes/epoptes#97] [Q&A]
I've successfully setup `epoptes-client` on all the student machines I'm working with; however, the instructors' machines run Manjaro 18.0.4 KDE (or i3) and I've not found packages for the `epoptes` server on those systems. Have I missed something?

**Comment by alkisg** [comment]
Hi, the work flow is:
- Upstream Epoptes developers write the code
- Distribution packagers provide the packages

It happens that Epoptes developers are also packagers for Debian and Ubuntu.

But for other distributions, you need to contact the distribution packagers, not the upstream Epoptes developers.

Maybe https://aur.archlinux.org/packages/epoptes/ can help, I don't know, I've never used arch-based distributions.

**Comment by seconddayout** [comment]
Ahh, I see. 'Preciate the explanation @alkisg.

And, yes, [the package you linked](https://aur.archlinux.org/packages/epoptes/) _is_ the thing to use. I came to learn so shortly after opening this issue. Being my first experience with an Arch based distro, I was unaware of AUR and its role/prominence in the Arch ecosystem. I'd expected epoptes to be accessible via pacman.

In any case, thanks.
